edb/server/net_worker.py

```diff
@@ -134,6 +135,10 @@ class ScheduledRequest:
     body: typing.Optional[bytes]
     headers: typing.Optional[list[dict]]
 
+    def __post_init__(self):
+        if self.body is not None:
+            self.body = base64.b64decode(self.body).decode('utf-8').encode()
```

Comment:
Ahhh, this is unfortunate but okay for now - we should figure out a better way to run queries internally other than `parse_execute_json()` I suppose